src/components/sections/Services.tsx

```diff
@@ -11,124 +11,18 @@ export class Services extends React.Component {
-							</div>
-						</div>
-
-						<div className="item">
-							<div className="single-services">
-								<div className="icon">
-									<i className="flaticon-shout" />
-								</div>
-								<h3>Marketing Solutions</h3>
-
-								<i className="flaticon-right link-btn" />
-
-								<Link to="#" className="link uk-position-cover" />
-							</div>
-						</div>
-
-						<div className="item">
-							<div className="single-services">
-								<div className="icon">
-									<i className="flaticon-analytics-1" />
-								</div>
-								<h3>Business Analytics</h3>
-
-								<i className="flaticon-right link-btn" />
-
-								<Link to="#" className="link uk-position-cover" />
-							</div>
-						</div>
-
-						<div className="item">
-							<div className="single-services active">
-								<div className="icon">
-									<i className="flaticon-ux-design" />
-								</div>
-								<h3>UX & UI Design</h3>
-
-								<i className="flaticon-right link-btn" />
-
-								<Link to="#" className="link uk-position-cover" />
-							</div>
-						</div>
-
-						<div className="item">
-							<div className="single-services">
-								<div className="icon">
-									<i className="flaticon-camera" />
-								</div>
-								<h3>Photography</h3>
-
-								<i className="flaticon-right link-btn" />
-
-								<Link to="#" className="link uk-position-cover" />
-							</div>
-						</div>
-
-						<div className="item">
-							<div className="single-services">
-								<div className="icon">
-									<i className="flaticon-project" />
-								</div>
-								<h3>Management</h3>
-
-								<i className="flaticon-right link-btn" />
-
-								<Link to="#" className="link uk-position-cover" />
-							</div>
-						</div>
-
-						<div className="item">
-							<div className="single-services">
-								<div className="icon">
-									<i className="flaticon-chat" />
-								</div>
-								<h3>Social Media</h3>
-
-								<i className="flaticon-right link-btn" />
-
-								<Link to="#" className="link uk-position-cover" />
-							</div>
-						</div>
-
-						<div className="item">
-							<div className="single-services">
-								<div className="icon">
-									<i className="flaticon-data" />
-								</div>
-								<h3>Data Science</h3>
-
-								<i className="flaticon-right link-btn" />
-
-								<Link to="#" className="link uk-position-cover" />
-							</div>
-						</div>
+						<Item title="Affidavits" iconName="flaticon-plan" />
+						<Item title="Business Contracts" iconName="flaticon-think" />
+						<Item title="Power of Attorney" iconName="flaticon-shout" />
+						<Item title="Apostille Services" iconName="flaticon-analytics-1" />
+						<Item title="Live Scan" iconName="flaticon-ux-design" />
+						<Item title="Document Translation" iconName="flaticon-camera" />
+						<Item title="Wills and Living Trusts" iconName="flaticon-project" />
+						<Item
+							title="Documentos y Trãmites Legales en Colombia"
```

Comment:
Not sure if you noticed but this service ends up being way too long and throws off the flow and styling of the services section as a whole. 

Let's replace it with "Colombia Legal" and plan to add more details about what this means in the modal or page that comes up after clicking on the service.